src/Compilers/VisualBasic/Portable/Symbols/SymbolExtensions.vb

```diff
@@ -415,5 +415,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Return vbSymbol
         End Function
 
+        <Extension>
+        Friend Function ContainingNonLambdaMember(member As Symbol) As Symbol
+            While member?.Kind = SymbolKind.Method AndAlso DirectCast(member, MethodSymbol).MethodKind = MethodKind.AnonymousFunction
```

Comment:
>member?.Kind = SymbolKind.Method AndAlso DirectCast(member, MethodSymbol).MethodKind = MethodKind.AnonymousFunction [](start = 18, length = 115)

Could you please verify that the use of the conditional access here doesn't produce less efficient IL, by comparison to an explicit null check? That could be the case given that VB uses three-valued logic.